packages/backend/src/services/ValidationService/ValidationService.ts

```diff
@@ -200,25 +205,26 @@ export class ValidationService {
                 [],
             );
 
-            const sortErrors = chart.metricQuery.sorts.reduce<
-                CreateValidation[]
-            >(
-                (acc, field) =>
-                    containsFieldId(
-                        acc,
-                        existingFieldIds,
-                        field.fieldId,
-                        `Sorting error: the field '${field.fieldId}' no longer exists`,
-                    ),
-                [],
-            );
+            const sortErrors = chart.metricQuery.sorts
+                .filter(
+                    (sort) =>
+                        filterTableCalculations(sort.fieldId) &&
+                        filterAdditionalMetrics(sort.fieldId),
+                )
+                .reduce<CreateValidation[]>(
+                    (acc, field) =>
+                        containsFieldId(
+                            acc,
+                            existingFieldIds,
+                            field.fieldId,
+                            `Sorting error: the field '${field.fieldId}' no longer exists`,
+                        ),
+                    [],
+                );
 
             /*
             // I think these are redundant, as we already check dimension/metrics
```

Comment:
should we just remove all the commented code below?

Reply:
not for now... I feel we might want to add it anyway, and it took me a while to generate those, specially the one about table calculations. For example, the method to filter table calculations was commented out and I have just recovered it.

Reply:
if you prefer, I can remove it and add a comment pointing to github, so we can easily track the code

Reply:
Cool, keep it @rephus, as long as we clean it up closer to the milestone completion then we're good! No action then, 👍